src/main/io/serial.c

```diff
@@ -437,9 +437,9 @@ serialPort_t *openSerialPort(
     serialPort_t *serialPort = NULL;
 
 #ifdef USE_SOFTSERIAL
-    if (((identifier == SERIAL_PORT_SOFTSERIAL1) || (identifier == SERIAL_PORT_SOFTSERIAL2)) && (baudRate > 19200)) {
+    if (((identifier == SERIAL_PORT_SOFTSERIAL1) || (identifier == SERIAL_PORT_SOFTSERIAL2)) && (baudRate > 57600)) {
```

Comment:
You could use constrain here.

However the 19200 limit is there for good reason; see https://github.com/betaflight/betaflight/pull/13303

Reply:
Smartport needs 57600 baud to work. Other protocols could be constraint.

Reply:
We now have the `ARMING_DISABLED_LOAD` flag. Aware of the disarm issue.